x-pack/plugins/security_solution/public/common/containers/sourcerer/index.test.tsx

```diff
@@ -411,6 +416,58 @@ describe('Sourcerer Hooks', () => {
         ]);
       });
     });
+
+    it('Should put any excludes in the index pattern at the end of the pattern list', async () => {
```

Comment:
If the order of the _excludes_ is slightly modified, as described in [this comment](https://github.com/elastic/kibana/pull/130069/files#r862253911), this test also verifies the `sortWithExcludesAtEnd` function sorts both the _includes_ and _excludes_.

Assuming that suggestion is implemented, consider making this explicit in the description by changing it to something like:

```
it('Should put any excludes in the index pattern at the end of the pattern list, and sort both the includes and excludes', async () => {
```